tests/e2e/e2e_k8s_ingress_http_test.go

```diff
@@ -26,6 +26,7 @@ var _ = OSMDescribe("HTTP ingress using k8s Ingress API",
 		It("allows HTTP ingress traffic", func() {
 			// Install OSM
 			installOpts := Td.GetOSMInstallOpts()
+			installOpts.EnableIngressBackendPolicy = false
 			installOpts.SetOverrides = []string{"OpenServiceMesh.featureFlags.enableIngressBackendPolicy=false"}
```

Comment:
What about the override in L30? Is that still required?

Reply:
Looks like we can get rid of this line from the test itself, but it will need to be added to common.go along with the other SetOverrides